### DISCUSSION TITLE
syntax error, unexpected ')' on Model Function #44

### DISCUSSION THREAD

**Discussion by dfrey382** [spatie/laravel-searchable#109] [Q&A]
`<?php

namespace Modules\Motor\Entities;

use Illuminate\Database\Eloquent\Model;
use Modules\Motor\Entities\Customer;
use Spatie\Searchable\Searchable;
use Spatie\Searchable\SearchResult;
class Vehicle extends Model implements Searchable
{
    //Table name
    protected $table = 'digital_lab_motor_vehicles';
    protected $fillable = [];

    //Customer relationship
    public function user()
    {
        return $this->belongsTo('Modules\Motor\Entities\Customer');
    }

    public function customer()
    {
        return $this->belongsTo(Customer::class, 'user_id');
    }

    //Quotations Relationship
    public function quotations()
    {
        return $this->hasMany('Modules\Motor\Entities\Quotation')->orderBy('created_at', 'desc');
    }

    //Premier Quotations Relationship
    public function premier_quotations()
    {
        return $this->hasMany('Modules\Motor\Entities\MotorPremierQuotation')->orderBy('created_at', 'desc');
    }

    public function getSearchResult(): SearchResult
    {

        return new SearchResult(
            $this,
            $this->veh_reg_number,
        );
    }
}`

**Comment by dfrey382** [comment]
> `<?php
> 
> namespace Modules\Motor\Entities;
> 
> use Illuminate\Database\Eloquent\Model;
> use Modules\Motor\Entities\Customer;
> use Spatie\Searchable\Searchable;
> use Spatie\Searchable\SearchResult;
> class Vehicle extends Model implements Searchable
> {
> //Table name
> protected $table = 'digital_lab_motor_vehicles';
> protected $fillable = [];
> 
> ```
> //Customer relationship
> public function user()
> {
>     return $this->belongsTo('Modules\Motor\Entities\Customer');
> }
> 
> public function customer()
> {
>     return $this->belongsTo(Customer::class, 'user_id');
> }
> 
> //Quotations Relationship
> public function quotations()
> {
>     return $this->hasMany('Modules\Motor\Entities\Quotation')->orderBy('created_at', 'desc');
> }
> 
> //Premier Quotations Relationship
> public function premier_quotations()
> {
>     return $this->hasMany('Modules\Motor\Entities\MotorPremierQuotation')->orderBy('created_at', 'desc');
> }
> 
> public function getSearchResult(): SearchResult
> {
> 
>     return new SearchResult(
>         $this,
>         $this->veh_reg_number,
>     );
> }
> ```
> 
> }`

the error occurs if in any controller I try to ;
$checkVehicle = Vehicle::where('veh_chasis_number', '=', $request->details['veh_chasis_number'])->first();

**Reply by muarachmann** [reply]
check your code, you have a syntax error somewhere. Probably you forgot to put a semicolon or parameter somewhere
